modular_skyrat/modules/supermatter_glow_component/glow.dm

```diff
@@ -0,0 +1,95 @@
+#define COMSIG_SM_PROCESS_ATMOS "atmos_process"
```

Comment:
Instead of undeffing this at the end of the file let other stuff use the signal aswell in the future, move it to a define file

Reply:
I don't think anything else will use it in the future are this is a modular part, but, maybe.